app/tests/testsv2/test_change_status.py

```diff
@@ -0,0 +1,124 @@
+import json
+from . import BaseClass
+
+
+class ParcelTestCase(BaseClass):
+    """This test case tets the parcel test cases"""
+    def test_admin_change_status(self):
+        """Test API if it adds a parcel(POST)"""
+        data = {"email": "admin@gmail.com",
+                "password": "admin",
+                "role": "admin"}
+        data2 = {"email": "langatchirchir@gmail.com",
+                 "password": "kevin12345",
+                 "role": "user"}
+        status = {"status": "deliver"}
+        res = self.client().post("api/v2/auth/signup", json=self.admin)
+        res2 = self.client().post("api/v2/auth/login", json=data)
+        data = json.loads(res2.get_data(as_text=True))
+        token_admin = data['data']['token']
+        res = self.client().post("api/v2/auth/signup", json=self.user)
+        res2 = self.client().post("api/v2/auth/login", json=data2)
+        data = json.loads(res2.get_data(as_text=True))
+        token_user = data['data']['token']
+        res = self.client().post("api/v2/parcels", json=self.parcel, headers=dict(Authorization="Bearer " + token_user))
+        data = json.loads(res.get_data(as_text=True))
+        order_id = data['parcel']['order_no']
+        res = self.client().put("api/v2/parcels/" + order_id + "/status", json=status,
+                                headers=dict(Authorization="Bearer " + token_admin))
+        self.assertEqual(res.status_code, 202)
+        self.assertIn("processing", str(res.data))
```

Comment:
res.data.status

Reply:
The res,data.status is returning an error